types/styled-system/styled-system-tests.tsx

```diff
@@ -273,7 +275,8 @@ interface GridComponentProps
         GridAutoColumnsProps,
         GridAutoRowsProps,
         GridTemplatesRowsProps,
-        GridTemplatesColumnsProps {}
+        GridTemplatesColumnsProps,
+        JustifyItemsProps {}
```

Comment:
This should probably be added to [`FlexComponentProps`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/247a03b2198d698e7283095f951aebf549705668/types/styled-system/styled-system-tests.tsx#L252)